website/source/intro/getting-started/build-image.html.md

```diff
@@ -12,9 +12,7 @@ description: |-
 # Build an Image
 
 With Packer installed, let's just dive right into it and build our first image.
-Our first image will be an [Amazon EC2 AMI](https://aws.amazon.com/ec2/) with
-Redis pre-installed. This is just an example. Packer can create images for [many
-platforms][platforms] with anything pre-installed.
+Our first image will be an [Amazon EC2 AMI](https://aws.amazon.com/ec2/) This is just an example. Packer can create images for [many platforms][platforms].
```

Comment:
let's try to wrap lines at 79 characters

Reply:
👍